server/sandbox.go

```diff
@@ -300,6 +300,8 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 		return nil, fmt.Errorf("failed to remove sandbox %s directory: %v", *sbID, err)
 	}
 
+	s.releasePodName(sb.name)
```

Comment:
we should also remove the sandbox ID from `s.state.sandboxes[id]` otherwise it can panic when listing - fixed into https://github.com/kubernetes-incubator/cri-o/pull/61